pom.xml

```diff
@@ -102,6 +102,7 @@
     <org.apache.poi.version>3.13</org.apache.poi.version>
     <org.apache.poi-ooxml.version>3.13</org.apache.poi-ooxml.version>
     <org.apache.tika.version>1.5</org.apache.tika.version>
+    <com.drewnoakes.version>2.6.2</com.drewnoakes.version>
```

Comment:
Why do not use thae latest version 2.14.0 ?

Reply:
this is the old PR, the new PR is https://github.com/Meeds-io/maven-depmgt-pom/pull/11

Reply:
Could you close this PR ?

Reply:
done!